backend/src/main/java/com/ambacam/service/ActionService.java

```diff
@@ -0,0 +1,115 @@
+package com.ambacam.service;
+
+import com.ambacam.exception.ResourceBadRequestException;
+import com.ambacam.exception.ResourceNotFoundException;
+import com.ambacam.model.Action;
+import com.ambacam.repository.ActionRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+@Transactional(rollbackFor = Exception.class)
+public class ActionService {
+	@Autowired
+	private ActionRepository actionRepository;
+
+	/**
+	 * Create an action
+	 * 
+	 * @param action The new action to create
+	 * 
+	 * @return The new action created
```

Comment:
Please remove the comments after @return

Reply:
Please do the same for other functions

Reply:
This one is also fine. Normally, it should be done like that. But because of laziness, we use to omit it. But having it is a good practice.